android-json-form-wizard/src/main/java/com/vijay/jsonwizard/interactors/JsonFormInteractor.java

```diff
@@ -104,7 +111,7 @@ protected void registerWidgets() {
         map.put(JsonFormConstants.SPACER, new ComponentSpacerFactory());
         map.put(JsonFormConstants.NATIVE_EDIT_TEXT, new NativeEditTextFactory());
         map.put(JsonFormConstants.TIME_PICKER, new TimePickerFactory());
-        map.put(JsonFormConstants.REPEATING_GROUP, new RepeatingGroupFactory());
```

Comment:
Avoid re-formatting these since it's inconsistent with the formatting in the rest of the class.